templates/taxi/driver_list.html

```diff
@@ -20,6 +24,12 @@ <h1>Driver List
         <td>{{ driver.first_name }}</td>
         <td>{{ driver.last_name }}</td>
         <td>{{ driver.license_number }}</td>
+          <td>
+              <a href="{% url "taxi:driver-update" pk=driver.id %}">UPDATE License</a>
+          </td>
+          <td>
+              <a href="{% url "taxi:driver-delete" pk=driver.id %}">DELETE Driver</a>
+          </td>
```

Comment:
these links should be in "driver_detail.html", "On the driver detail page add buttons that lead to the driver's license updating page and driver deletion page."